content/zh-cn/security-chaos-engineering.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 安全混沌工程
+status: Completed
+category: 概念
+---
+
+## 是什么
+
+安全混沌工程（SCE） 是基于[混沌工程](/zh-cn/chaos-engineering/)的学科。 SCE 是指在分布式系统上执行主动安全实验，以建立对系统抵御动荡和恶意条件能力的信心。安全混沌工程师使用科学方法来不断实现这一点，这些方法包括稳态、假设、持续验证、经验教训和缓解实施。
+
+## 解决的问题
+
+[网站稳定性工程师](/site-reliability-engineering/)（SRE） 和网络安全工程师的主要职责是尽快恢复服务，来达到零停机时间和最小化业务影响的目标。 SRE 和网络安全工程师共同处理故障前和故障后的各种意外情况。大多数安全问题很难快速发现和修补，这将影响应用程序或系统的功能。此外，在开发阶段通常很难发现安全事件。
```

Comment:
```suggestion
[网站稳定性工程师](/zh-cn/site-reliability-engineering/)（SRE） 和网络安全工程师的主要职责是尽快恢复服务，来达到零停机时间和最小化业务影响的目标。 SRE 和网络安全工程师共同处理故障前和故障后的各种意外情况。大多数安全问题很难快速发现和修补，这将影响应用程序或系统的功能。此外，在开发阶段通常很难发现安全事件。
```

Reply:
`site-reliability-engineering` has not translate so far. When `dev-zh` merge into `main` this will cause a 404 problem  if I use `/zh-cn/site-reliability-engineering/` instead of `/site-reliability-engineering/`.